ddev/ddev
# fix: Drupal 10 and Drupal 11 settings have migrated, set state_cache

## The Issue

Drupal 10 and Drupal 11 settings have migrated slightly.

The most important thing is $state_cache in Drupal 10.3

## How This PR Solves The Issue

* Update Drupal 11 settings to current defaults
* Update Drupal 10 settings to current defaults and set `$settings['state_cache'] = TRUE;` in the settings.ddev.php
* Use Drupal 10 settings if version can't be detected
* Add comment to all Drupal and Backdrop settings.php/settings.ddev.php to show where they have come from.
* Removed the "drupal" settings.php/settings.ddev.php. Instead we'll use drupal10 by default if can't detect the actual version.


## Manual Testing Instructions

Spin up Drupal 11 and Drupal 10 sites. See that the warning about state_cache does appear.
